### DISCUSSION TITLE
Create New Field, Save and Add Another

### DISCUSSION THREAD

**Discussion by richhayler** [craftcms/cms#11244] [Ideas] [answered]
_I previously raised this suggestion but with Craft 4 having just launched and people taking the new version for a spin I thought it relevant to bring this up again for consideration. 

Similar to the option when creating a new entry, it would be nice to have a dropdown menu as part of the new field “save” button to “save and add a new field” (ideally defaulting to the selected field group). This would be useful when initially setting up a new section for example and creating a bunch of fields.

As an added bonus, it would be great to add a new field based on the previously selected field type and selected options. 

Duplicate: https://github.com/craftcms/cms/discussions/8671

**Comment by brandonkelly** [comment, marked as answer]
Please don’t intentionally create duplicate suggestions.

**Reply by richhayler** [reply]
Apologies, noted.
